### PR TITLE
heyo

### PR DESCRIPTION
**Pull request by bkern24**
heyo

### CODE REVIEW

**Comment by bkern24** [comment]
Eugeneeeeeeeeeeeeeeeee!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Comment by bkern24** [comment]
For realz?

**Comment by bkern24** [comment]
Come on man let me merge

**Comment by bkern24** [comment]
wow....

**Comment by bkern24** [comment]
ridiculous

**Comment by bkern24** [comment]
fine......

File: core/assets/Lonely_Trees.tmx [hunk 1]
```diff
@@ -0,0 +1,269 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<map version="1.0" orientation="orthogonal" renderorder="right-down" width="100" height="100" tilewidth="32" tileheight="32" nextobjectid="1">
+ <tileset firstgid="1" name="Tileset" tilewidth="32" tileheight="32">
+  <image source="generic_platformer_tiles.png" width="1024" height="768"/>
+  <tile id="69">
+   <properties>
+    <property name="collidable" value="1"/>
```
Comment (sturmE) [inline core/assets/Lonely_Trees.tmx:7]: Why did you choose this value?

File: core/assets/Lonely_Trees.tmx [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+   </properties>
+  </tile>
+  <tile id="293">
+   <properties>
+    <property name="collidable" value="1"/>
+   </properties>
+  </tile>
+  <tile id="300">
```
Comment (sturmE) [inline core/assets/Lonely_Trees.tmx:40]: WTF, tile 300?

File: core/assets/Lonely_Trees.tmx [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+  <tile id="419">
+   <properties>
+    <property name="collidable" value="1"/>
+   </properties>
+  </tile>
+  <tile id="449">
+   <properties>
+    <property name="collidable" value="1"/>
+   </properties>
+  </tile>
```
Comment (sturmE) [inline core/assets/Lonely_Trees.tmx:104]: Spelling mistake

File: core/assets/Lonely_Trees.tmx [hunk 4]
```diff
@@ -0,0 +1,269 @@
...
...6lJNr33PU/rSa7+O20nDyK94fkVzk/XhvO9LT58Sv0Ev61LTua9pE3vPU8auF5hznl/ht+Gc87RFPTA3eK3SFkPUM9Kn1G/Qs0pzMGIde89zxqwXqIvpsevED+caxxSf8d7jWNj9kvWAp1VJPeO6V5KDXp+oZx37E4d96kZE04Xzg9e6qMfUiaGX85FX2f3ceiDSqiTGUd3zvLeN+D57utHTqC9NF9YD89db/3Ed5JxHr+JvQz/34tw239vWGpHPch7n+vLaNj1W1Xl/z82jPkTN7Yhz2fOs3J6Z9wccW9OvJB+8+OEYWKMoF7y5EXkn6496oL/PO+haSq+eYy6g5/NczK1nc2seL4Z9PAXbwHvoi56PcU1ro+mB/r5sNBmTXM95fmK8+bvsm0vXQfWzuT1cn7006sAx9/qb0Thza6+apgf6+6rRxOj1tay60avnHBucJ5jr/DvH+tncHq40L6I84Th7/aEPlqy9jFjT0VeM0f4uxxXQq+deLbS85lzvkvMlLNG1VPv9YFyYrx736GjtYU23vcIYY92Dtr16zj5unoRji9ax/wK9XELOgiPqYXuF+QicQduYN5h/vM61MbF/lXh73/9Mxmgj127bmHkNgmss2yssR+C8aZfzhtfVXjx4z5Hzxr712VsrdfU+HovFNRqnHfF9z6+Oq7+1ZDUCl82R8wb3H6gJ7idMB8ztnDf2JbZjc6GPL3E70Thn1fn5h8+jHkP+C0FeT7yReLNaX6stSY8pjS2an7nakdOrjV7bXfwRdezDyK+G/BeCfJj4KPGw8vOG9+djeM6YNYU9rMQfu+YUMvKrPrke6bSofE/gb8K15xDPKYlZl3rS1R/RW7sy8quueY6+xPcWdD6v1v+Hse8Ye08wVF97v82DuC/LK/SfrkfWY57hi8SXcI6+xM8u6HxZrf8PY/3n8hh/R8d9p60hWubqlTfOoUfWYxnQdLideIfuLYJ3+PrK6cP6z+Wxl9PR8TJYOrbSY7S+ZR3saPdfNXF/Te9F15nHE3Eo36Y4HlVn69rDdP4uHd831+z6h8JYf6z+D34u4EX1/QXifuSwvv5rC2J0VfXAo1F6bIceyo/t0kP5cTl6fMpQemxWj6+J3xK/J/5I/EnXpIf0kB7SQ3psJ6XH9vEytJAe20fpcbE86Pi/nvSQHleJ0mN7WGshPbaHl63HH6OarbU=
```
Comment (sturmE) [inline core/assets/Lonely_Trees.tmx:266]: O nice. Did you compute that all by yourself?

**Pull request merged**